docs/GettingStarted.md

```diff
@@ -240,7 +162,9 @@ defining your Grape application:
     require 'grape'
```

Comment:
can't reach the previous line but we should change the part:
```
To activate your integration, use the ``patch_module`` function
```

since we're not using the `patch_module` anymore as a public API.

Reply:
(Fixed)